twake/backend/node/src/services/channels/entities/channel.ts

```diff
@@ -0,0 +1,46 @@
+import { Entity, Column, ObjectID, ObjectIdColumn, PrimaryColumn } from "typeorm";
+
+export enum VisibilityEnum {
+  PRIVATE = "private",
+  PUBLIC = "public",
+  DIRECT = "direct"
+}
+
+@Entity({ name: "channels" })
+export class Channel {
+  // uuid-v4
+  @PrimaryColumn()
+  company_id: string;
+
```

Comment:
Shouldn't we use a special type for UUID-v4 instead of string ?

Reply:
it depends of the driver used. When really implementing it we will have to take care and it was not the goal of this issue